src/category_theory/functor.lean

```diff
@@ -80,8 +80,23 @@ infixr ` ⋙ `:80 := comp
 @[simp] lemma comp_obj (F : C ⥤ D) (G : D ⥤ E) (X : C) : (F ⋙ G).obj X = G.obj (F.obj X) := rfl
 @[simp] lemma comp_map (F : C ⥤ D) (G : D ⥤ E) (X Y : C) (f : X ⟶ Y) :
   (F ⋙ G).map f = G.map (F.map f) := rfl
+
+omit ℰ
+
+@[simp] protected lemma comp_id (F : C ⥤ D) : F ⋙ (𝟭 D) = F :=
```

Comment:
I’m pretty dubious about providing any public lemmas asserting equalities between functors. Of course they are true, but it is playing with fire to assert equalities rather than natural isomorphisms, and I’d prefer to not encourage people to go down this route. I agree having `Cat` is useful, but my preference would be to hide these lemmas in that file, leaving only the natural isomorphisms (which exist already) in the main API.

Reply:
Done.